web-app/messages/en.json

```diff
@@ -119,7 +119,8 @@
             },
             "Username": {
               "min": "User Name must be at least 2 characters.",
-              "max": "User Name must be maximum 50 characters."
+              "max": "User Name must be maximum 50 characters.",
```

Comment:
I think spelling wise Username or username should be correct?

Reply:
f0b52c47be7dbdeff814096bb8102d0c3e5ab8e0